subxt/src/storage/utils.rs

```diff
@@ -37,3 +42,48 @@ pub(crate) fn storage_address_root_bytes<Address: StorageAddress>(addr: &Address
     write_storage_address_root_bytes(addr, &mut bytes);
     bytes
 }
+
+/// Strips the first 16 bytes (8 for the pallet hash, 8 for the entry hash) off some storage address bytes.
+pub fn strip_storage_addess_root_bytes(
```

Comment:
Do we want to publically expose this and below?

We expose this `utils` module publically at the moment by the looks of it, which was pointless because the fns were all `pub(crate)` so probably a relic from before!

I'd propose just no longer exposing the `utils` mod at all and then we can turn the `pub(crate)`s into `pub`s (unless there is some reason to expose these functions publically)